CHANGELOG

```diff
@@ -19,6 +48,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.3.9] 2025-01-07
 
+## Added
+
+- ItemAvailability class and updated access messaging
```

Comment:
All these changes in the changelog are correct, but I'm curious, why are they showing up as "new"? Not a blocker for release.